### DISCUSSION TITLE
Integration in Nextcloud

### DISCUSSION THREAD

**Discussion by linux-lukas** [OpenSlides/OpenSlides#5869] [Ideas]
In my view, it could be useful for OpenSlides to have an integration with Nextcloud in order to further expand FLOSS Nextcloud as a central collaboration component.

**Comment by linux-lukas** [comment]
Perhaps it would be also cool for OpenSlides to notify participants via Nextcloud of changes to the agenda, new motions, new amendment motions, etc.

**Comment by linux-lukas** [comment]
In the context of openslides version 4, I think it would make sense to force this idea.

**Comment by ostcar** [comment]
I have no idea how a Software can be integrated with Nextcloud. I also don't know if only a Software written in PHP can be integrated with NextCloud or how the API of Nextcloud looks like.

If you are interested in researching this questions, curious about the results.

**Comment by GabrielInTheWorld** [comment]
There are NextCloud clients for different languages like [Python](https://pypi.org/project/nextcloud-api-wrapper/) or [NodeJS](https://www.npmjs.com/package/nextcloud-node-client) available. An integration could be made, but this is not on our roadmap for version 4.0.
